apps/bfd-model/bfd-model-rif-samples/src/main/resources/rif-static-samples/sample-hicn-mult-bene-beneficiaries.txt

```diff
@@ -4,7 +4,7 @@ INSERT|5678342019|MO|123|12345|17-MAR-1981|1|1|1|1|N|20|0|0|543217066U|Doe|John|
 INSERT|-123456|MO|123|12345|17-MAR-1981|1|1|1|1|N|20|0|0|543217066N|DoeNullReferenceYear|John|A|3456789N|17-Mar-1981||4|5|6|7|8|A|B|C|D|4|19631003|5|AA|AA|AA|AA|AA|AA|AA|AA|AA|AA|AA|AA|4|4|AA|AA|AA|AA|AA|AA|AA|AA|AA|AA|AA|AA|5|C|C|C|C|C|C|C|C|C|C|C|C|1|1|1|1|1|1|1|1|1|1|0|0|H1225|H1225|H1225|H1225|H1225|H1225|H1225|H1225|H1225|H1225|0|0|003|003|003|003|003|003|003|003|003|003|||00|00|00|00|00|00|00|00|00|00|||S4607|S4607|S4607|S4607|S4607|S4607|S4607|S4607|S4607|S4607|S4607|0|003|003|003|003|003|003|003|003|003|003|003||000|000|000|000|000|000|000|000|000|000|000||Y|Y|Y|Y|Y|Y|Y|Y|Y|Y|Y|N|**|**|**|**|**|**|**|**|**|**|**|**|00|00|01|01|05|06|07|08|01|02|00|00|123 MAIN ST|APT ABC|NOT REAL|FAKE ADDRESS|ANOTHER FAKE ADDRESS|MORE FAKE ADDRESS|PODUNK|IA|123456789|30-JUL-2020|31-JUL-2020|12345|17-Mar-2020|17-JUN-2020|17-JUL-2021|17-AUG-2022|17-FEB-2021|17-NOV-2022
-INSERT|1234|MO|123|12345|17-MAR-1981|1|1|1|1|N|20|0|0|SAMEHICN|Doe|John|A|SAMEMBI|17-Mar-1981|2019|4|5|6|7|8|A|B|C|D|4|19631003|5|AA|AA|AA|AA|AA|AA|AA|AA|AA|AA|AA|AA|4|4|AA|AA|AA|AA|AA|AA|AA|AA|AA|AA|AA|AA|5|C|C|C|C|C|C|C|C|C|C|C|C|1|1|1|1|1|1|1|1|1|1|0|0|H1225|H1225|H1225|H1225|H1225|H1225|H1225|H1225|H1225|H1225|0|0|003|003|003|003|003|003|003|003|003|003|||00|00|00|00|00|00|00|00|00|00|||S4607|S4607|S4607|S4607|S4607|S4607|S4607|S4607|S4607|S4607|S4607|0|003|003|003|003|003|003|003|003|003|003|003||000|000|000|000|000|000|000|000|000|000|000||Y|Y|Y|Y|Y|Y|Y|Y|Y|Y|Y|N|**|**|**|**|**|**|**|**|**|**|**|**|00|00|01|01|05|06|07|08|01|02|00|00|123 MAIN ST|APT ABC|NOT REAL||ANOTHER FAKE ADDRESS|MORE FAKE ADDRESS|PODUNK|IA|123456789|30-JUL-2020|31-JUL-2020|12345|17-Mar-2020|17-JUN-2020|17-JUL-2021|17-AUG-2022|17-FEB-2021|17-NOV-2022
+INSERT|98547|MO|123|12345|17-MAR-1981|1|1|1|1|N|20|0|0|SAMEHICN|Doe|John|A|SAMEMBI|17-Mar-1981|2019|4|5|6|7|8|A|B|C|D|4|19631003|5|AA|AA|AA|AA|AA|AA|AA|AA|AA|AA|AA|AA|4|4|AA|AA|AA|AA|AA|AA|AA|AA|AA|AA|AA|AA|5|C|C|C|C|C|C|C|C|C|C|C|C|1|1|1|1|1|1|1|1|1|1|0|0|H1225|H1225|H1225|H1225|H1225|H1225|H1225|H1225|H1225|H1225|0|0|003|003|003|003|003|003|003|003|003|003|||00|00|00|00|00|00|00|00|00|00|||S4607|S4607|S4607|S4607|S4607|S4607|S4607|S4607|S4607|S4607|S4607|0|003|003|003|003|003|003|003|003|003|003|003||000|000|000|000|000|000|000|000|000|000|000||Y|Y|Y|Y|Y|Y|Y|Y|Y|Y|Y|N|**|**|**|**|**|**|**|**|**|**|**|**|00|00|01|01|05|06|07|08|01|02|00|00|123 MAIN ST|APT ABC|NOT REAL||ANOTHER FAKE ADDRESS|MORE FAKE ADDRESS|PODUNK|IA|123456789|30-JUL-2020|31-JUL-2020|12345|17-Mar-2020|17-JUN-2020|17-JUL-2021|17-AUG-2022|17-FEB-2021|17-NOV-2022
```

Comment:
Some context here, previously a test was supposed to ensure that an MBI that existed in both bene_history and bene tables returned the bene table data. However, the data was such that the two mbi entries had different bene ids, which threw an exception. This prevented the test from testing the case it mentioned (and the assertion was also asserting this error instead of the test case mentioned.)

Fixed this by adjusting the data and writing up a correct test case for this in the new E2E suite.